### PR TITLE
[SCB-1610] add updateInstanceStatus to ServiceRegistryClient.java

### PR DESCRIPTION
**Pull request by merryzsp**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean install -Pit` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by liubao68** [comment]
I create a issue to track this and updated this title. Can you do the following ?
1. format the code using /etc foler code template
2. rebase and push again to help better merge the PR (This can help mantain all commits or we have to do a merge).

**Comment by merryzsp** [comment]
> I create a issue to track this and updated this title. Can you do the following ?
> 
> 1. format the code using /etc foler code template
> 2. rebase and push again to help better merge the PR (This can help mantain all commits or we have to do a merge).

我没有理解到你的意思  是说代码没有格式化么？还是说要做什么其他操作？

**Comment by liubao68** [comment]
> > I create a issue to track this and updated this title. Can you do the following ?
> > 
> > 1. format the code using /etc foler code template
> > 2. rebase and push again to help better merge the PR (This can help mantain all commits or we have to do a merge).
> 
> 我没有理解到你的意思 是说代码没有格式化么？还是说要做什么其他操作？

是的，少量代码没有格式化，建议适用项目提供的模板格式化一下。 第二个问题是指提交PR的分支可能不是基于master拉的，或者中间隔了几个commit，虽然也可以通过merge的方式合并，但是会丢失commit信息，可以进行一下rebase，保证可以在合入的时候，保留commit信息。 
image

File: service-registry/src/main/java/org/apache/servicecomb/serviceregistry/client/http/ServiceRegistryClientImpl.java [hunk 1]
```diff
@@ -901,4 +902,37 @@ public ServiceCenterInfo getServiceCenterInfo() {
...
+      if (LOGGER.isDebugEnabled()) {
+        LOGGER.debug("update status of microservice instance: {}", status);
+      }
+      String url = String.format(Const.REGISTRY_API.MICROSERVICE_INSTANCE_STATUS, microserviceId, microserviceInstanceId);
+      if (StringUtils.isEmpty(status)) {
+        LOGGER.debug("empty status");
+        return false;
+      }
+      Map<String, String[]> queryParams = new HashMap<>();
+      queryParams.put("value",new String[]{status});
```
Comment (liubao68) [inline service-registry/src/main/java/org/apache/servicecomb/serviceregistry/client/http/ServiceRegistryClientImpl.java]: code not well formated

Comment (merryzsp) [reply service-registry/src/main/java/org/apache/servicecomb/serviceregistry/client/http/ServiceRegistryClientImpl.java]: fixed

**Pull request merged**